### PR TITLE
Prac 08 feedback

### PR DESCRIPTION
**Pull request by JoshuaDelosSantos**
Hi @danbrooks784 👹

Please check my naming and SRP 😊

Thanks! 🤓

### CODE REVIEW

File: prac_08/squaring.kv [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+            multiline: False
+        Button:
+            text: 'Square'
+            # the following line specifies the function in the app class to call when the button is pressed
+            on_press: app.handle_calculate(input_number.text)
+        Label:
+            id: output_label
+            font_size: 48
+            color: (7, 0, 1, 1)
+    BoxLayout:
```
Comment (danbrooks784) [inline prac_08/squaring.kv]: You don't need to add a BoxLayout if there is only one element in it

File: prac_08/convert_miles_km.py [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+        self.root = Builder.load_file('convert_miles_km.kv')
+        self.message = "Type in the field & press convert"
+        return self.root
+
+    def handle_calculate(self):
+        """Handle calculation for converting miles to kilometres."""
+        try:
+            result = float(self.root.ids.input_number.text) * MILE_TO_KILOMETRE_RATE
+            self.root.ids.output_label.text = str(result)
+        except ValueError:
```
Comment (danbrooks784) [inline prac_08/convert_miles_km.py]: You can use a function instead of doing these ValueError exception checks multiple times 👍

**Review by danbrooks784 — COMMENTED** [review]
Really good work! 👍

File: prac_08/convert_miles_km.py (hunk 2 above)
Comment (JoshuaDelosSantos) [reply prac_08/convert_miles_km.py]: Good thinking SRP!

File: prac_08/squaring.kv (hunk 1 above)
Comment (JoshuaDelosSantos) [reply prac_08/squaring.kv]: 🫡

**Pull request merged**